test-configs.yaml

```diff
@@ -381,6 +381,12 @@ device_types:
       - blacklist: *allmodconfig_filter
       - blacklist: {defconfig: ['multi_v7_defconfig+CONFIG_LKDTM=n']}
 
+  cubietruck:
+    name: 'sun7i-a20-cubietruck'
```

Comment:
I think the upstream device type name is `cubietruck`:
https://git.lavasoftware.org/lava/lava/blob/master/lava_scheduler_app/tests/device-types/cubietruck.jinja2#L1

It's true that using the dt name is better, but that would need to be done using the new LAVA device type alias feature.  Otherwise, I think it has to be `cubietruck`.

Reply:
With your change, kcicore does not generate a job, where I need to set the DT name for let kcicore find it?

Reply:
The linux DT name is sun7i-a20-cubietruck

Reply:
oh i just saw dtb: